File: API/Core/Entities/Feedback.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace API.Core.Entities
{
    public class Feedback : BaseEntity
    {
        public string? FullName { get; set; }
        public string? PhoneNumber { get; set; }
        public string? Email { get; set; }
        public string? Surburb { get; set; }
        public string? TimeToCall { get; set; }
        public string? Message { get; set; }
        public DateTime ReceivedTime { get; set; } = DateTime.Now;
        public string? AppUserId { get; set; }
        public AppUser? AppUser { get; set; }
    }
}

File: client/src/app/shared/models/feedback.ts
export interface IFeedback {
    fullName: string;
    phoneNumber: string;
    email: string;
    surburb: string;
    timeToCall: string;
    message: string;
    receivedTime: Date;
    appUserId: string;
    id: number;
}

export class Feedback {
    fullName: string = '';
    phoneNumber: string = '';
    email: string = '';
    surburb: string = '';
    timeToCall: string = '';
    message: string = '';
    receivedTime: Date = new Date();
    appUserId: string = '';
    id: number = 0;
}